Car.h

```diff
@@ -1,9 +1,13 @@
 #pragma once
-
-struct Car : public Vehicle
+#include "Vehicle.h"
+struct Car :  Vehicle
 {
+    ~Car() override = default;
     Car(const std::string& s);
+    Car(const Car& c) = default;
+    Car& operator=(const Car& ref) = default;
     void closeWindows();
-
-    void tryToEvade();
+    void setSpeed(int s) override;
```

Comment:
remove this function, it is not needed and should not be overridden.
Don't change code that does not need to be changed.
- [ ] revise

Reply:
it is not obvious to me why this did not need to be overriden